src/lib.rs

```diff
@@ -1215,4 +1235,24 @@ mod tests {
                      "  bar"];
         assert_eq!(dedent(&add_nl(&x)), add_nl(&y));
     }
+
+    #[test]
+    fn multiline() {
+        assert_eq!(fill("1 3 5 7\n1 3 5 7",11),"1 3 5 7\n1 3 5 7");
```

Comment:
Could you run `rustfmt` on the file? I believe it would insert a space after the `,`. Thanks!